weaviate/values.yaml

```diff
@@ -37,6 +37,27 @@ args:
 #   - "-c"
 #   - "ulimit -n 65535 && /bin/weaviate --host 0.0.0.0 --port 8080 --scheme http --config-file /weaviate-config/conf.yaml"
 
+
+# it is possible to change the sysctl's 'vm.max_map_count' using initContainer for Weaviate,
+# the init Container runs before Weaviate Container and sets the value for the WHOLE node
+# to the one provided below.
+# it is possible to run additional initContainer before Weaviate is up and running. You can specify the
+# containers as a list, exactly how they are defined in a kubernetes manifest:
+#   https://kubernetes.io/docs/concepts/workloads/pods/init-containers/
+initContainers:
+  sysctlInitContainer:
+    enabled: true
+    sysctlVmMaxMapCount: 524288
+    image:
+      registry: docker.io
+      repo: alpine
+      tag: latest
+      pullPolicy: IfNotPresent
+  
+  extraInitContainer: {}
+  # - image: some-image
+  #   name: some-name
```

Comment:
Here just some cosmetic changes, maybe let's change that first would be the name? wdyt?
```
  # - name: some-name
  #   image: some-image
```
and do we want to add here also additional settings, roughy something like this, wdyt?
```
#      securityContext: #optional
#         runAsUser: 0 #optional
 #        privileged: true #optional
#        imagePullPolicy: some-pull-policy #optional
  #      command: some-comman #optional
```

Reply:
I thought about this but it just follows the official `initContainer` structure and is not Weaviate specific that is why I did not add any additional configurations.

Reply:
ok, agreed